awscli/clidocs.py

```diff
@@ -159,6 +161,14 @@ def doc_options_start(self, help_command, **kwargs):
     def doc_option(self, arg_name, help_command, **kwargs):
         doc = help_command.doc
         argument = help_command.arg_table[arg_name]
+        if is_streaming_blob_type(argument.argument_model):
+            doc.style.start_note()
+            msg = ("This argument is of type: streaming blob. "
+                   "Its value must be the path to a file "
+                   "(e.g. ``path/to/file``) and must **not** "
+                   "be prefixed with ``file://`` or ``fileb://``")
+            doc.include_doc_string(msg)
```

Comment:
Was there a particular reason `include_doc_string()` was used instead of `writeln()`? The main reason I ask is the `include_doc_string()` method is generally only used when we are dealing with XML/HTML based docstrings that we need to convert to RSt.

Reply:
Ah I see. No reason other than it works. Changing to `writeln()`.